phylogenetic/rules/construct_phylogeny.smk

```diff
@@ -13,16 +13,20 @@ rule tree:
         alignment="results/{virus}/aligned.fasta",
     output:
         tree="results/{virus}/tree_raw.nwk",
+    threads:
+        workflow.cores * 0.5
```

Comment:
Why `* 0.5`?

Reply:
This repo has four different virus builds. Empirically, running on my laptop, plain `workflow.cores` ends up as effectively blocking on one build at a time, which seems to effectively take a little bit longer. Using `workflow.cores * 0.5` gives enough space that this _particular_ step is faster than running single-threaded, but the other rules can also continue to execute "around" it.

Reply:
Thanks, that's helpful context. The increased run time with more threads per rule seems like the same thing that's being discussed in https://github.com/nextstrain/dengue/pull/105. I think what was done there makes sense: optimize defaults for wherever the automation is running (i.e. GitHub Actions).